sktime/performance_metrics/detection/__init__.py

```diff
@@ -7,6 +7,8 @@
 from sktime.performance_metrics.detection._randindex import RandIndex
 from sktime.performance_metrics.detection._ts_auprc import TimeSeriesAUPRC
 
+# TimeSeriesAUPRC is a utility for computing AUPRC, not strictly a detection metric,
```

Comment:
this is from another pull request? It does not have to do anything with `plot_windows`